banners/desktop/styles/styles.scss

```diff
@@ -1,4 +1,6 @@
 // This is the file where we import the theme-specific component styles
+$font: 'Iowan Old Style', 'Sitka Text', 'Cambria', serif;
```

Comment:
I fixed up the font stack in the theme for banner 11, but this came back in from 10.

You can delete this line and remove the `with ( $font: $font )` from the Message and KeenSlider imports